crypto/vm/debugops.cpp

```diff
@@ -105,6 +105,37 @@ int exec_dump_value(VmState* st, unsigned arg) {
   return 0;
 }
 
+int exec_dump_string(VmState* st) {
+  VM_LOG(st) << "execute STRDUMP";
+  if (!vm_debug_enabled) {
+    return 0;
+  }
+
+  Stack& stack = st->get_stack();
+  auto item = stack[0];
```

Comment:
Check depth before accessing the stack.
Don't use magic number (`.is(4)`). `.as_slice()` returns null if item is not a slice, so it can be used here.